scripts/_utils/utils.py

```diff
@@ -84,6 +91,50 @@ def verify_mimetype(file_url, mimetype_string, local=False):
         return False
 
 
+def is_ffmpeg_compatible(file_url) -> bool:
+    command = "ffmpeg -y -v error -i " + file_url + " /tmp/verify-ffmpeg.mp4"
+    err = subprocess.run(command.split(" "), capture_output=True).stderr
+    return err == b''
+
+
+def verify_media_integrity(file_url, mime, local) -> Tuple[bool, Union[str, None], bool]:
+    """
+    Verifies and fixes media
+    :returns: status and file path and local
+    """
+    try:  # test if input is image
+        im = Image.open(file_url)
+    except PIL.UnidentifiedImageError:  # input is not image
+        return True, file_url, local
+
+    if mime == 'image/gif':
```

Comment:
This shouldn't be in the verify_media_integrity method, it should be in it's own "process_gif" method

Reply:
fixed in [ab06ea3](https://github.com/timberline-secondary/hackerspace-scripts-2/pull/48/commits/ab06ea3ff440c835606d11ff5f6e2405f1d65554)